practice-typescript/src/scripts/controllers/user.ts

```diff
@@ -42,6 +43,11 @@ class UserController {
   handleGetById = async (id: string): Promise<UserModel> => {
     return await this.service.getById(id);
   };
+
+  handleSearchByName = async (name: string): Promise<UserModel> => {
```

Comment:
Can users have the same name? Or your search result will be an array, not single user?

Reply:
fixed